### PR TITLE
Es upload

### PR DESCRIPTION
**Pull request by thomascpan**

### CODE REVIEW

File: dags/common/utils.py [hunk 1]
```diff
@@ -254,6 +243,17 @@ def build_case_report_json(xml_path: str) -> dict:
     return case_report
 
 
+def validate_case_report(case_report: dict) -> bool:
```
Comment (carolynyen) [inline dags/common/utils.py:246]: If this is the same as checking "if case_report.get('pmID')", is there some advantage to defining it?

File: script/elasticsearch_script.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk, scan
+import json
+
+es = Elasticsearch(
```
Comment (carolynyen) [inline script/elasticsearch_script.py:5]: Do we still need this script?

File: dags/common/utils.py (hunk 1 above)
Comment (thomascpan) [reply dags/common/utils.py:246]: In case we need other validations. It only has 1 right now.

File: script/elasticsearch_script.py (hunk 2 above)
Comment (thomascpan) [reply script/elasticsearch_script.py:5]: Keep it for now just in case. This is to edit existing ones.

**Pull request merged**